packages/functional-tests/tests/oauth/signinTokenCode.spec.ts

```diff
@@ -36,61 +33,51 @@ test.describe('severity-2 #smoke', () => {
     };
     /* eslint-enable camelcase */
 
-    test.beforeEach(async ({ target }, { project }) => {
+    test.beforeEach(async ({ target, emails }, { project }) => {
       // The `sync` prefix is needed to force confirmation.
-      email = `sync${Math.random()}@restmail.net`;
-      emailUserCreds = await target.createAccount(email, password);
-    });
-
-    test.afterEach(async ({ target }) => {
-      // Cleanup any accounts created during the test
-      try {
-        await target.auth.accountDestroy(
-          email,
-          password,
-          {},
-          emailUserCreds.sessionToken
-        );
-      } catch (e) {
-        // ignore
-      }
+      const [syncEmail] = emails;
+      emailUserCreds = await target.createAccount(syncEmail, password);
```

Comment:
Could remove the assignment I think.
```suggestion
      await target.createAccount(syncEmail, password);
```